pages/common/xh.md

```diff
@@ -0,0 +1,28 @@
+# xh
+
+> XH: Friendly and fast tool for sending HTTP requests.
+> More information: <https://github.com/ducaale/xh>.
+
+- Send a GET request:
+
+`xh {{httpbin.org/get}}`
+
+- Send a POST request with JSON body: `{"name": "ahmed", "age": 25}`:
+
+`xh post {{httpbin.org/post}} {{name=john}} {{age:=25}}`
+
+- Send a GET request with querystring id=5&sort=true:
+
+`xh get {{httpbin.org/get}} {{id==5}} {{sort==true}}`
+
+- Send a GET request and include a header named x-api-key with value 12345:
+
+`xh get {{httpbin.org/get}} {{x-api-key:12345}}`
+
+- Send a PUT request and pipe the result to less:
+
+`xh put {{httpbin.org/put}} {{id:=49}} {{age:=25}} | less`
+
+- Download and save to res.json:
+
+`xh -d {{httpbin.org/json}} -o {{res.json}}`
```

Comment:
```suggestion
- Make a GET request and save the response body to a file:

`xh --download {{httpbin.org/json}} --output {{path/to/file}}`
```
Also note that we tend towards using the long options of commands when possible as they're usually self explanatory and invaluable to the beginner.